configs/pie-components-config/package.json

```diff
@@ -4,9 +4,15 @@
   "main": "index.js",
   "type": "module",
   "description": "Shared configuration files for PIE components",
+  "files": [
+    "custom-elements-manifest.config.js"
+  ],
   "dependencies": {
     "deepmerge-ts": "5.1.0"
   },
+  "peerDependencies": {
```

Comment:
I'm not sure if this is correct, but figured I'd add it. Basically this needs to be added as a devDep to our package in order for the `create:manifest` script to run successfully, due to its use in the shared config.